HackBeanpot/www-2019
# New nav and footer!

## What this does

* Adds responsive nav and footer
* Creates a new component called `DynamicLink` which takes a `to` path as a prop and will dynamically return a `<Link>` for internal page routes or an `<a>` tag that opens in a new tab for external links
* Updates home page "About" section with illustrations
* Uses another Material Design icon package

**Nav:**
<img width="1280" alt="Screen Shot 2019-11-03 at 9 30 26 PM" src="https://user-images.githubusercontent.com/22566949/68096828-5d3c9b00-fe81-11e9-8a8d-7812ff426574.png">
<img width="240" alt="Screen Shot 2019-11-03 at 9 31 32 PM" src="https://user-images.githubusercontent.com/22566949/68096830-6168b880-fe81-11e9-82e9-d48b013e9f09.png">
<img width="240" alt="Screen Shot 2019-11-03 at 9 31 39 PM" src="https://user-images.githubusercontent.com/22566949/68096831-63cb1280-fe81-11e9-89d2-f12e762b8f65.png">

**Footer:**
<img width="1280" alt="Screen Shot 2019-11-03 at 9 30 34 PM" src="https://user-images.githubusercontent.com/22566949/68096847-80ffe100-fe81-11e9-984f-9d9862b26958.png">
<img width="240" alt="Screen Shot 2019-11-03 at 9 31 48 PM" src="https://user-images.githubusercontent.com/22566949/68096849-852bfe80-fe81-11e9-8287-65dc00ee8804.png">

## How to test
1. Run `npm install`
2. Make sure everything looks good at all screen sizes and that the mobile nav works as expected

## TODO

- [x] Figure out link colors
- [x] Make `NavMobile` a functional component
